tests/everflow/everflow_test_utilities.py

```diff
@@ -721,14 +721,35 @@ def send_and_check_mirror_packets(self,
         src_port_set = set()
         src_port_metadata_map = {}
 
-    
+
         if 't2' in setup['topo']:
             if valid_across_namespace is True:
                 src_port_set.add(src_port)
-                src_port_metadata_map[src_port] = (None, 1)
+                if duthost.facts['switch_type'] == "voq":
+                    src_port_metadata_map[src_port] = (None, 1)
+                    if self.mirror_type() != "egress":  # no egress route on the other node/namespace
+                        src_port_set.add(dest_ports[0])
+                        src_port_metadata_map[dest_ports[0]] = (setup[direction]["egress_router_mac"], 1)
```

Comment:
why we have to egress_router_mac ?

Reply:
We did not add egress_router_mac.  It was there already